xtest/sdk/java/cli.sh

```diff
@@ -57,7 +57,7 @@ if [ "$1" == "encrypt" ]; then
   args+=(--kas-url=$KASURL)
 
   if [ "$USE_ECDSA_BINDING" == "true" ]; then
-    args+=(--ecdsa-binding "true")
+    args+=(--ecdsa-binding)
```

Comment:
should this part be commented out until implemented in the cli again to avoid false positives?

Reply:
Leaving it here allows us to use xtest for testing the feature during development. If we see problems on release branches of the java sdk, we can cherry-pick the commit that disabled this in the cmdline